zpay32/invoice_internal_test.go

```diff
@@ -1,11 +1,149 @@
 package zpay32
 
 import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"reflect"
 	"testing"
+	"time"
 
 	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/ltcsuite/ltcd/chaincfg/chainhash"
+	"github.com/roasbeef/btcd/btcec"
+	"github.com/roasbeef/btcd/chaincfg"
+	"github.com/roasbeef/btcutil"
+	"github.com/roasbeef/btcutil/bech32"
 )
 
+var (
+	testMillisat24BTC    = lnwire.MilliSatoshi(2400000000000)
```

Comment:
Can you move the test data back to where it was? Since they are in the same package now, it should be accessible from there, and we'll minimize the diff :)